pubsub/google/cloud/pubsub/topic.py

```diff
@@ -435,12 +462,27 @@ def publish(self, message, **attrs):
 
         :type attrs: dict (string -> string)
         :param attrs: key-value pairs to send as message attributes
+
+        :rtype: None
```

Comment:
There are several places where the function _does_ return in order to stop control flow, which makes the linter require `:rtype:`.